.github/workflows/build-centos-pypa-rust.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Centos Pypa Aarch64 & x86_64 Rust Image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'centos-pypa-aarch64/*'
+      - 'centos-pypa-x86_64/*'
+      - '.github/workflows/build-centos-pypa-rust.yml'
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 12 * * 5'
+
+concurrency:
+  # SHA is added to the end if on `main` to let all main workflows run
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ github.event_name }}-${{ github.ref == 'refs/heads/main' && github.sha || '' }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [aarch64, x86_64]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+# shellcheck disable=SC2296,SC2086
+      - name: Set docker tag to build
+        if: matrix.arch == 'aarch64'
+        id: dockertag
+        run: echo "dockertag=ghcr.io/chia-network/centos-pypa-${{ matrix.arch }}:${{ github.sha }}" >>$GITHUB_OUTPUT
+
+      - name: Build Docker Container
+        if: matrix.arch == 'aarch64'
+        uses: docker/build-push-action@v2
+        with:
+          context: ./centos-pypa-rust-aarch64
+          file: ./centos-pypa-rust-aarch64/Dockerfile
+          push: true
+          tags: ${{ steps.dockertag.outputs.dockertag }}
+# shellcheck disable=SC2296,SC2086
```

Comment:
You wanna indent this at the same level as the line it's above, and it also needs to go directly above the line that shellcheck is arguing over

Reply:
It's currently a few lines above where it should be, and not indented correctly